server/services/change_gating/bitbucket_adapter.py

```
@@ -0,0 +1,605 @@
+        uuids = set()
+        for client in {id(self._post): self._post, id(self._read): self._read}.values():
+            try:
+                me = client.get_current_user()
+                if isinstance(me, dict) and not me.get("error"):
+                    uuid = me.get("uuid")
+                    if uuid:
+                        uuids.add(uuid)
+            except Exception as exc:
+                logger.warning(
+                    "[ChangeGating:BB] identity lookup failed: %s", type(exc).__name__
+                )
```

Ruff (0.16.1)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)